explanation/security/security_suggestions.md

```diff
@@ -0,0 +1,177 @@
+   * For more details on basic access control, see our {ref}`guide on user management <user-management>`.
+
+### Network security
+
+1. **Use a firewall**. In Ubuntu, the uncomplicated firewall (`ufw`) tool is
+   used to configure firewalls. `ufw` is a wrapper around the `iptables` utility
+   (which experienced system admins may prefer to use directly). To get started
+   with `ufw`, check out our {ref}`firewalls` guide.
+
+1. **Use the Secure Shell (SSH)** protocol to secure remote access. In Ubuntu,
+   this is managed through OpenSSH. For details on setting up OpenSSH, refer to
+   our {ref}`guide to OpenSSH <openssh-server>`. 
+
+### Physical security
+
+There are also steps you can take to protect the physical security of your
+system. These how-to guides will help you set up these additional precautions:
+
+* {ref}`Smart card authentication <smart-card-authentication>`.
+* {ref}`Smart card authentication with SSH <smart-card-authentication-with-ssh>`.
+* {ref}`Console security <console-security>`.
+
+
+(advanced-security)=
+## Suggestions for complex setups
+
+The following section will help direct you to the security-related packages for
+which we provide documentation. For more discussion about advanced security
+considerations, refer to the [Ubuntu Security](https://ubuntu.com/security)
+documentation. 
+
+### Advanced Access Control
+
+1. **Lightweight Directory Access Protocol (LDAP)** is the usual way to gate
+   access control for larger or more complex setups. In Ubuntu, this is
+   implemented through OpenLDAP. Refer to our
+   {ref}`introduction to OpenLDAP <introduction-to-openldap>`
+   for more details, or see our section
+   {ref}`on how to set up OpenLDAP <how-to-openldap>`.
+1. **Kerberos** is a network authentication protocol that provides identity
+   verification for distributed environments, commonly used in enterprise
+   systems. Learn more in our
+   {ref}`introduction to Kerberos <introduction-to-kerberos>`, or see our
+   section on how to {ref}`set up and use Kerberos <how-to-kerberos>`.
+1. **System Security Services Daemon (SSSD)** is a collection of daemons that
+   handle authentication, authorisation and user/group information from
+   disparate network sources. It integrates with OpenLDAP, Kerberos, and
+   Active Directory as we discuss in more detail in our
+   {ref}`introduction to SSSD <introduction-to-network-user-authentication-with-sssd>`
+   or get started setting it up with our
+   {ref}`how-to section <how-to-network-user-authentication-with-sssd>`.
+
+### Virtual Private Networks (VPNs)
+
+1. **WireGuard VPN**
+
+   * {ref}`Introduction to WireGuard VPN <introduction-to-wireguard-vpn>`
+   * {ref}`How to set up WireGuard VPN <how-to-wireguard-vpn>`
+
+1. **OpenVPN**
+   
+   * {ref}`About OpenVPN clients <openvpn-client-implementations>`
+   * {ref}`How to install OpenVPN <install-openvpn>`
+
+### Security of communications
+
+1. **TLS/SSL** for secure communication
```

Comment:
I think we need here a short "why you'd want transport security" in general, and can then refer to those examples - because in reality it is always different and depends on the solution they are setting up.

Remiding users to use some transport level secuirty is fair without explaining all potential options.
Just be open that many stacks offer it, some even try to use it by default, but even then it requires certificate/secrets management which is why it almost always needs the system administrator to get it to be really good.